test/fixtures/package_todo.yml

```diff
@@ -2,7 +2,7 @@
 buyers:
   "::Buyers::Document":
     violations:
-    - dependency
+    - some_checker_type
```

Comment:
This was originally used to test when a dependency violation turned into a privacy violation. I changed it to privacy originally and then realized we could use any arbitrary string to represent a different checker's violation.